syntax = "proto3";

service Showtime {
  rpc GetListTimes(Empty) returns (stream MoviesSchedules) {};
  rpc GetMovieByDate (Date) returns (Movies);
}
message Date {
  string date = 1;
}

message Movies {
  repeated string movies = 1;
}

message MoviesSchedules {
  string date = 1;
  repeated string movies = 2;
}

message Empty {
}